tests/graph/test_basic.py

```diff
@@ -501,6 +527,16 @@ def test_graph_inputs():
     assert res_list == [r3, r1, r2]
 
 
+def test_required_graph_inputs():
+    x = pt.fscalar()
```

Comment:
Test is fine, but it's not using any of the stuff you added above

Reply:
Yes those did not work as expected and I realised that I will not have to add those altogether.